src/main/scala/com/scylladb/migrator/Validator.scala

```diff
@@ -119,11 +122,12 @@ object RowComparisonFailure {
               leftTtl  = left.getLongOption(name)
```

Comment:
Oops, this (and the line below) should be `leftWritetime`.
```suggestion
              leftWritetime  = left.getLongOption(name)
```

Reply:
Whoops, fixed!